ibl_to_nwb/NwbToOne/nwb_to_ibl_metadata.py

```diff
@@ -1,14 +1,29 @@
-from pynwb import NWBHDF5IO
+import json
+from oneibl.one import ONE
+from pynwb import NWBFile, NWBHDF5IO
 import uuid
+import h5py
 import sys
 from datetime import datetime
-from .field_map import *
+from .field_map import field_map_nwbfile, field_map_session_data, \
+    field_map_subject,field_map_IBL_subject
 from ndx_ibl_metadata import IblProbes
 from copy import copy, deepcopy
 import h5py
 
 
 def _convert_numpy_to_python_dtype(dict_to_convert_in):
+    """
+    Converts numpy dtypes like np.int*, np.float* to python native int, float
+    Conversion useful to store data in .json format which supports only native
+    datatypes.
+    Parameters
+    ----------
+    dict_to_convert_in: dict
+    Returns
+    -------
+    dict_to_convert_out: dict
+    """
     dict_to_convert_out = dict()
     for key, val in dict_to_convert_in.items():
         if 'numpy' in str(type(val)):
```

Comment:
I think there should be a better way to do this with `isinstance`

Reply:
correct, using `np.generic` is for all scalar numpy types. Will change